pypgstac/pypgstac/migrations/pgstac.0.6.9-0.6.10.sql

```diff
@@ -0,0 +1,42 @@
+SET SEARCH_PATH to pgstac, public;
+set check_function_bodies = off;
+
+CREATE OR REPLACE FUNCTION pgstac.get_queryables(_collection_ids text[] DEFAULT NULL::text[])
+ RETURNS jsonb
+ LANGUAGE plpgsql
+ STABLE
+AS $function$
+BEGIN
+    -- Build up queryables if the input contains valid collection ids or is empty
+    IF (select array_agg(id) from collections) @> _collection_ids OR
```

Comment:
Not sure if this is the right implementation.

Reply:
Pushed a change that just switches this logic up with a bit more efficient approach using exists. Nice work with stringing together some PLPGSQL @mmcfarland !

Reply:
Nice, thanks!